src/core/public/http/http_service.ts

```diff
@@ -50,9 +52,23 @@ export class HttpService implements CoreService<HttpSetup, HttpStart> {
 
   public setup({ injectedMetadata, fatalErrors }: HttpDeps): HttpSetup {
     const opensearchDashboardsVersion = injectedMetadata.getOpenSearchDashboardsVersion();
+    let workspaceBasePath = '';
+    const plugins = injectedMetadata.getPlugins();
```

Comment:
Yes but actually what I want to do here is to make the logic here exactly the same as before when workspace is not enabled. But it turns out that it will introduce another indirect dependency on the workspace plugin. I will revert to previous version, in which it will try to construct workspaceBasePath no matter workspace is enabled or not, and let server side to gives a `404` when workspace is disabled, do you think that would be better?